packages/components/src/Popover/Popover.test.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useRef } from "react";
+import {
+  RenderResult,
+  act,
+  cleanup,
+  fireEvent,
+  render,
+} from "@testing-library/react";
+import { Popover } from ".";
+import { PopoverProps } from "./Popover";
+
+afterEach(cleanup);
+
+let rendered: RenderResult;
+const content = "Test Content";
+
+const PopoverTestComponent = (props: PopoverProps) => {
+  const checkboxRef = useRef();
+  return (
+    <>
+      <div ref={checkboxRef}></div>
+      <Popover {...props} attachTo={checkboxRef}>
+        {props.children}
+      </Popover>
+    </>
+  );
+};
+
+it("should render a Popover with the content and dismiss button", async () => {
+  const handleClose = jest.fn();
+
+  await act(async () => {
+    rendered = render(
+      <PopoverTestComponent
+        attachTo={undefined}
```

Comment:
What does it do if you pass `undefined` here? Is this something it should support?

Reply:
I was having some difficulty creating my HOC `PopoverTestComponent`.  Hooks can only be created in the component and I built the HOC to accept PopoverProps which unfortunately requires `attachTo`. So I just pass undefined there. 

```
const PopoverTestComponent = (props: PopoverProps) => {
  const checkboxRef = useRef();
  return (
    <>
      <div ref={checkboxRef}></div>
      <Popover {...props} attachTo={checkboxRef}>
        {props.children}
      </Popover>
    </>
  );
};
```

Would there be another way?